### PR TITLE
useNativeDriver warnings fixed

### PR DESCRIPTION
**Pull request by Harin329**

### CODE REVIEW

File: Swiper.js [hunk 1]
```diff
@@ -195,7 +195,11 @@ class Swiper extends Component {
       })
     }
 
-    return Animated.event([null, this.createAnimatedEvent()])(
+    return Animated.event([null, this.createAnimatedEvent()], {
+      listener: () => null,
```
Comment (webraptor) [inline Swiper.js:199]:
```suggestion
```

No need to add empty optional.

**Review by webraptor — CHANGES_REQUESTED** [review]
Please bump package version (patch).

Thanks

**Comment by Harin329** [comment]
Version bumped! Thanks for picking this up from the old repo!

File: Swiper.js [hunk 2]
```diff
@@ -195,7 +195,10 @@ class Swiper extends Component {
       })
     }
 
-    return Animated.event([null, this.createAnimatedEvent()])(
+    return Animated.event([null, this.createAnimatedEvent()], {
+      useNativeDriver: false
+    })
+    (
```
Comment (webraptor) [inline Swiper.js:201]:
```suggestion
    return Animated.event([null, this.createAnimatedEvent()], { useNativeDriver: false }) (
```

File: Swiper.js [hunk 3]
```diff
@@ -195,7 +195,7 @@ class Swiper extends Component {
       })
     }
 
-    return Animated.event([null, this.createAnimatedEvent()])(
+    return Animated.event([null, this.createAnimatedEvent()], { useNativeDriver: false }) (
```
Comment (webraptor) [inline Swiper.js:198]:
```suggestion
    return Animated.event([null, this.createAnimatedEvent()], { useNativeDriver: false })(
```

**Review by webraptor — APPROVED** [review]

**Comment by webraptor** [comment]
> Version bumped! Thanks for picking this up from the old repo!

Glad to help out!

**Pull request merged**